app/api/orders.py

```diff
@@ -86,6 +87,16 @@ def before_create_object(self, data, view_kwargs):
                 raise ConflictException({'pointer': '/data/relationships/attendees'},
                                         "Attendee with id {} does not exists".format(str(ticket_holder)))
 
+        ticket = db.session.query(Ticket).filter_by(
```

Comment:
Okay, I think you are confused about what we are trying to achieve here. We have already implemented restriction on orders which have at least one free ticket placed by an unverified user. But we want to restrict the **orders** which are completely free (not only one ticket free). You are still restricting orders which have even a single free ticket and other paid tickets